SmellDetector/SmellModel/Reports/PartialSmellDetectionReport.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace SmellDetector.SmellModel.Reports
+{
+    public class PartialSmellDetectionReport
+    {
+        public Dictionary<string, List<Issue>> PartialReport { get; set; }
```

Comment:
A field called PartialReport in a class that is essentially called PartialReport is not a good name 🙂 

Following the previous renaming, we can call this CodeItemIssues.